src/test/java/org/apache/maven/shared/dependency/analyzer/DefaultProjectDependencyAnalyzerTest.java

```diff
@@ -302,17 +303,44 @@ public void testMultimoduleProject()
         assertEquals( expectedAnalysis, actualAnalysis );
     }
 
+    public void testTypeUseAnnotationDependency()
+            throws TestToolsException, ProjectDependencyAnalyzerException
+    {
+        // java.lang.annotation.ElementType.TYPE_USE introduced with Java 1.8
+        Assume.assumeTrue(SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_1_8));
```

Comment:
Assumptions doesn't work with methodName based tests like PlexusTestCase still does.
See https://builds.apache.org/job/maven-box/job/maven-dependency-analyzer/job/MSHARED-674_Detect_Java_8_type_annotations/1/

For know I'd suggest to simple do a return when this value is false, but keep the code once PlexusTestCase is upgraded.

Reply:
Thanks. I've changed that.